src/Components/Aspire.Azure.Data.Tables/AspireTablesExtensions.cs

```diff
@@ -114,7 +116,7 @@ private sealed class TableServiceComponent : AzureComponent<AzureDataTablesSetti
                 return !string.IsNullOrEmpty(connectionString) ? new TableServiceClient(connectionString, options) :
                     cred is not null ? new TableServiceClient(settings.ServiceUri, cred, options) :
                     new TableServiceClient(settings.ServiceUri, options);
-            }, requiresCredential: false);
```

Comment:
What is the impact of changing this? I see that it's always `true` in `AddClient` methods

Reply:
We shouldn't be changing this. We need to keep the current behavior.

Reply:
I fixed most changes back to the way they were. I missed the EventHubs ones, but I need to step away. Can you take care of that? I can take care of it when I get back too.